test_runner/regress/test_safekeeper_migration.py

```diff
@@ -132,6 +123,82 @@ def expect_fail(sk_set: list[int], match: str):
     expect_fail([sk_set[0], decom_sk], "decomissioned")
 
 
+def test_safekeeper_migration_common_set_failpoints(neon_env_builder: NeonEnvBuilder):
```

Comment:
great work, love this test

Reply:
Thanks)
For the context: it's not part of this PR, it was a part of
- https://github.com/neondatabase/neon/pull/12528